cmake/FindOpenEXR.cmake

```diff
@@ -198,6 +173,34 @@ if(EXISTS "${OpenEXR_INCLUDE_DIR}/OpenEXRConfig.h")
   set(OpenEXR_VERSION ${OpenEXR_VERSION_MAJOR}.${OpenEXR_VERSION_MINOR})
 endif()
 
+if(${OpenEXR_VERSION_MAJOR} MATCHES "3")
+  set(_OPENEXR_COMPONENT_LIST OpenEXR Iex IlmThread)
+else()
+  set(_OPENEXR_COMPONENT_LIST IlmImf)
```

Comment:
Any reason why IlmImfUtil was removed?

Reply:
The architectural change of note in OpenEXR3 is that we reorganized all the libraries per community feedback.

First, we split out Imath to be a standalone library with no other dependencies; previously it depended on Iex for example.

Second, we consolidated the various OpenEXR image format libraries and the support they required, under an OpenEXR module.

Reply:
@meshula that makes sense thanks, but I'm referring to the branch where branch where `OpenEXR_VERSION_MAJOR` != 3. This used to import `IlmImfUtil` for EXR 2 and it's been removed for EXR 2. I don't think we're even using it but I don't see why it was removed. Just trying to minimise any behaviour changes if they're not necessary.

Reply:
I took it out originally because it wasn't used so seemed superfluous, but I've added it back now, along with the 3.x counterpart OpenEXRUtil.

Reply:
The houdini CI build is failing, and apparently it can't find IlmImfUtil.  Is it possible this is linking against OpenEXR built into Houdini that doesn't include IlmImfUtil? I'm not sure what's different in that build vs the others. I can't easily build the Houdini dependencies, if someone who can could possibly look into what's going on.

Reply:
Yes, I think that is the problem. As far as I can tell - the find_package in openvdb/cmd/CMakeLists did not previously request the IlmImfUtil component so adding it in is almost certainly what is causing the Houdini CI to fail. However, all I think Nick was suggesting was to leave IlmImfUtil as a possible component in cmake/FindOpenEXR.cmake, even though it doesn't look like anything is actually using it.